lib/Monads/nondet/Nondet_VCG.thy

```diff
@@ -6,16 +6,17 @@
  *)
 
 theory Nondet_VCG
-imports
-  Nondet_Lemmas
-  WPSimp
+  imports
```

Comment:
when did we start indenting imports? I know the autoindenter likes to do it, but I'm not sure that that's our policy. @lsf37 ?

Reply:
I noticed that most of the files in `nondet` already had indented imports so I decided to go with the autoindenter and make them consistent. I don't know if we have an official policy, but I'd also be happy to go in the opposite direction and make them consistent with the rest of the repository.

Reply:
We probably shouldn't fight the auto indenter on this. We used to not indent them, but I'd be happy with indented import statements (and I think I have done that a few times myself now)